### DISCUSSION TITLE
BECMI?

### DISCUSSION THREAD

**Discussion by becherbrook** [decheine/complete-compendium#215] [Ideas]
Are there any plans to add all the BECMI monsters? It's my understanding it was quite easy to cross-pollinate with AD&D, and while a lot of them I can see are adapted/updated in the Mystara section there's definitely some missing from the Creature Catalogue (1993) eg. giant sand spider.

**Comment by decheine** [comment]
I like what you've brought up, I wasn't aware of the Creature Catalog until now, but I've found the pdf of the most updated Creature Catalog (1993) 9438 DMR2. There are a lot of great monster descriptions in here, and I think the Compendium ought to include these in some capacity. The Mystara ones (mostly from 2524) were already present in the data, and although many lack some paragraphs they always have all of the elements of the standard 2e statblock. 
These monsters, however, have:


```
Climare/Terrain
Frequency/Rarity
Armor Class:
Hit Dice:
Move:
Attacks:
Damage:
No. Appearing:
Save As:
Morale:
Treasure Type:
Intelligence:
Alignment
XP Value:
```

Missing some standard "columns" like Special Attacks/Defenses, Diet, Activity Cycle, Number of attacks, Thac0 for some reason, and magic resistance. The new "Save As:" field interests me, and could be something we retroactively create for existing monsters. 

What I may end up doing is labeling these monsters specially like "(Creature Catalog)" and creating new monsters entries for Creature Catalog monsters that have all the standard fields, like Climate and Thac0, filled out manually. Basically doing what Mystara did in 2524 but it's myself or someone else.  

Another option that would take less effort is: pick out all of the unique monsters in the Creature Catalog that don't exist in any form in the 
Compendium and make new monsters entries for them where I add missing info manually. 

EDIT: Made some edits on rereading, these monsters do all in include Terrain and Frequency/"Rarity" explicitly.

**Reply by decheine** [reply]
THAC0 can be calculated directly from HD. 

Diets are described in the paragraphs so the statblock value could be assigned from that, taking examples from existing monsters for the exact verbiage.

Special Attacks and Defenses are also described but with no exact verbiage so I would have to make it up.

MR could also be assumed to be N/A unless otherwise stated in the text. 

Activity Cycle may be a bit tricky. id have to look for "nocturnal" and such in each monster text.

I'm starting to see the evidence of an editing decision that was made to cut these  to instead put them in the paragraphs.

Also, another quirk, their alignments are the old-school Lawful, Neutral, Chaotic. Probably could just add neutral to the end of all of them. But I may preserve it as is, it's a quirk of editing/publishing.

So the plan for this would be to mark monsters that already exist and ones that are unique to the Creature Catalog, and then add new unque monsters and just source the ones that already exist. Could also add monsters that are variants of specific existing monsters to the existing monsters page - doing this will depend on how explicit the reference in the Creature Catalog monster text is. 

The "_Monster Type_" field is also interesting, and something I may want to add to all monsters, but later. Will still include it at the start of each entry text like in the book, but won't include it in the statblock.

The way the Compendium is set up there can be extra fields to a statblock, like in some Birthright monsters with extra Birthright stats. So that's what I'll do with the Save As field. Other existing monsters could benefit from this as well if their text says something special about saves, I know they exist.
